core/example/03_struct_of_arrays/soa_example.cpp

```diff
@@ -0,0 +1,142 @@
+/****************************************************************************
+ * Copyright (c) 2018 by the Cabana authors                                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cabana_Core.hpp>
+
+#include <iostream>
+
+//---------------------------------------------------------------------------//
+// SoA example.
+//---------------------------------------------------------------------------//
+void soaExample()
+{
+    /*
+      Cabana structs-of-arrays (SoAs) are groups of Tuples with the data
+      specified in an order such that the tuple index is stride-1 in
+      memory. This results in each dimension of each multidimensional tuple
+      member being stored in a contiguous memory block the size of the number
+      of tuples. The number of tuples in an SoA is defined as the "vector
```

Comment:
"number of tuples" may be "number of each tuple member"?

Reply:
Each member is one component of a tuple - so in this case the array size or vector length can also be viewed as the number of tuples in the SoA

Reply:
It cause some confusion to me because a tuple is an object that could hold one or many "particles". Are you using "particles" and "tuples" interchangeably?